package/kernel/mt76/Makefile

```diff
@@ -13,6 +13,7 @@ PKG_SOURCE_VERSION:=22b690334c0f49b11534cc2e331c9d5e17c4a0bc
 PKG_MIRROR_HASH:=ff5e563935919d2e40c1e7254ef3bc06f7ecc5e69f8ddd12903e8f5de942d630
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
+PKG_USE_NINJA:=0
```

Comment:
Why is this needed?

Reply:
I explained in the commit message.

mt76 is a kernel package mainly which uses make to build. The kernel stuff fails to build without this.